go.mod

```diff
@@ -1,26 +1,77 @@
-module "https://github.com/stormforger/cli"
```

Comment:
this change was necessary as the compile didn't seem to recognize itself correctly and downloaded the github repo for building everything else.